frontend/public/components/utils/resource-log.tsx

```diff
@@ -239,6 +341,17 @@ export const ResourceLog: React.FC<ResourceLogProps> = ({
     }
   }, [status, totalLineCount, previousTotalLineCount, bufferSize]);
 
+  //Check to see if previous log exists
+  React.useEffect(() => {
+    if (resource.kind === 'Pod') {
+      setLogType(LOG_TYPE_CURRENT); // Set back to viewing current log when switching resources or containers
```

Comment:
I believe this will switch the log type back to current when the pod status updates since we're watching the pod. Even if `resource` changes, it might be a status update rather than a totally different resource.

Reply:
Good catch, you are right.  Changed the code so the log type is reset back to `current` only if the container name changes (and if the resource.kind is `Pod`).